packages/components/Button/src/Button.types.ts

```diff
@@ -17,6 +18,11 @@ export interface ButtonCoreTokens extends LayoutTokens, FontTokens, IBorderToken
    */
   iconColor?: ColorValue;
 
+  /**
+   * Ripple color for Android.
+   */
+  rippleColor?: ColorValue;
+
```

Comment:
Any chance this can move to `IColorTokens` ? Looks like that comes from the file `color-tokens.ts` and you could probably copy what's being done for `IForegroundColorTokens/foregroundColorTokens` and make a `RippleColorTokens`. Or add it to `IBackgroundColorTokens if that's more appropriate. 

Not blocking though, and could be a followup

Reply:
Taking this offline